test/extreme.test.js

```diff
@@ -4,6 +4,7 @@ var test = require('tap').test
 var os = require('os')
 var writeStream = require('flush-write-stream')
 var spawn = require('child_process').spawn
```

Comment:
The scripts passed into `spawn`, even with `{stdio: [0, 1, 2]}`, are not receiving the actual process streams. Thus, the check for a tty is failing (after having been corrected to check properly). Do you have any suggestions for spawning the test script with a tty? Forking results in the same issue.